google/cloud/opentelemetry/internal/time_series.h

```diff
@@ -72,8 +77,13 @@ std::vector<google::monitoring::v3::TimeSeries> ToTimeSeries(
 
 /**
  * Convert from OpenTelemetry metrics to Cloud Monitoring protos.
+ *
+ * We return a vector of requests, because Cloud Monitoring accepts at most 200
```

Comment:
Leave the literal "200" to just the constant declaration?

Reply:
Yeah you are right. `200` should really only show up in one place. Done.